Lanchonete/LanchoneteTests/Controller/Presenter/ConfirmationViewPresenterTests.swift

```diff
@@ -0,0 +1,87 @@
+    }
+    
+    func test_GIVEN_a_menu_sandwich_WHEN_I_click_on_the_sandwich_THEN_I_should_get_the_ingredient_list() {
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        XCTAssertEqual(confirmationViewPresenter.getIngredientList(), "Ingredients: Hamburguer, Cheese, Bacon.")
+    }
+    
+    func test_GIVEN_a_customized_sandwich_WHEN_I_click_on_the_sandwich_THEN_I_should_see_the_name_X_Custom() {
+        let sandwichManager = SandwichManager()
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        XCTAssertEqual(confirmationViewPresenter.getSandwichName(), "X-Custom")
+    }
+    
+    func test_GIVEN_a_customized_sandwich_WHEN_I_click_on_the_sandwich_THEN_I_should_see_no_ingredients_yet() {
+        let sandwichManager = SandwichManager()
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        XCTAssertEqual(confirmationViewPresenter.getIngredientList(), "No Ingredients yet! 😳")
+    }
+    
+    func test_GIVEN_a_menu_sandwich_on_the_confirmation_screen_WHEN_I_click_on_the_increment_button_THEN_the_quantity_should_be_increased_by_one() {
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        
+        XCTAssertEqual(confirmationViewPresenter.incrementQuantity(), "2")
+    }
+    
+    func test_GIVEN_a_menu_sandwich_on_the_confirmation_screen_and_the_quantity_is_1_WHEN_I_click_on_the_decrement_button_THEN_the_quantity_should_be_decremented_by_one() {
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        XCTAssertEqual(confirmationViewPresenter.decrementQuantity(), "0")
+    }
+    
+    func test_GIVEN_a_menu_sandwich_on_the_confirmation_screen_WHEN_the_quantity_is_0_and_the_I_decrement_the_quantity_THEN_the_quantity_should_be_0() {
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        confirmationViewPresenter.quantity = 0
+        
+        XCTAssertEqual(confirmationViewPresenter.decrementQuantity(), "0")
+    }
+    
+    func test_GIVEN_a_menu_sandwich_WHEN_I_click_on_the_sandwich_THEN_I_should_see_the_specified_image() {
+        let sandwichManager = SandwichManager(with: .xBacon)
+        
+        let confirmationViewPresenter = ConfirmationViewPresenter(sandwichManager: sandwichManager)
+        
+        XCTAssertEqual(confirmationViewPresenter.getSandwichImage(), UIImage(named: "X-Bacon"))
```

Comment:
According to my previous comment, I believe that if you remove the UIImage handling from your manager, you wouldn't need to import and test UIKit in your unit tests.
The advantage of it is that you could unit test your code without the need of a simulator, like we do with our Presentation/Core/Data packages in Agent.

Reply:
I also removed this, can you check again?